tests/ci/commit_status_helper.py

```diff
@@ -488,4 +500,50 @@ def trigger_mergeable_check(commit: Commit, statuses: CommitStatuses) -> None:
     description = format_description(description)
 
     if mergeable_status is None or mergeable_status.description != description:
-        set_mergeable_check(commit, description, state)
+        set_mergeable_check(commit, description, state, hide_url)
+
+
+def trigger_a_sync_check(sync_commit: Commit, upstream_commit: Commit) -> None:
+    """calculate and update StatusNames.SYNC of the upstream commit"""
+    # Check the local JOBS_REQUIRED_FOR_SYNC and calculate the cumulative upstream
+    sync_statuses = get_commit_filtered_statuses(sync_commit)
+    required_sync_checks = [
+        status for status in sync_statuses if status.context in JOBS_REQUIRED_FOR_SYNC
+    ]
+
+    success = []
+    fail = []
+
+    # We don't change the StatusNames.SYNC of the sync_commit, only check it
+    for status in required_sync_checks:
+        if status.state == SUCCESS:
+            success.append(status.context)
+        else:
+            fail.append(status.context)
+
+    state: StatusType = SUCCESS
+
+    if success:
+        description = ", ".join(success)
+    else:
+        description = "awaiting job statuses"
+
```

Comment:
> In sync PR mergeable check status is set to FAIL
It will be done automatically. I plan to add the `A Sync` to the sync_pr too

> All jobs in Sync PR CI are finished with OK
We don't want all of them so far. That's why JOBS_REQUIRED_FOR_SYNC is used. As well as we use REQUIRED_CHECKS.

And since `StatusNames.SYNC` there, it will be considered too.

But the main thing you could miss, the Mergeable Check must be triggered for the upstream repo. That's why there so many `get_filtered_statuses` for different places.